### ISSUE TITLE
Config version: desktop.nix, de.nix

### ISSUE THREAD

**Issue by s0me1newithhand7s** [s0me1newithhand7s/myNixConf#3]
# Purpose
issue for my work on testing desktop version with different DE.
# Syntaxis
- no amd.nix/nvidia.nix bc it will be on VM.
- edits in config with screen and code.
- source to manual like nixos wiki.
# TODO
1. GNOME - ✅ 
2. Plasma (both 5 and 6) - ✅
3. Xfce4 - ✅
4. end tests, leave issue opened - ✅

**Comment by s0me1newithhand7s** [comment]
# Issue 2 in tree #3
### Config 
- Oracle VM VBox Manager
- toggled `sddm.enable` and `plasma5.enable` to `true`; minor edits in `desktop,nix`
- https://nixos.wiki/ and https://search.nixos.org/packages?channel=unstable
### Result / Error
no build errors, no syntaxis errors, no toggle errors;
KDE5 works fine, not as smooth as i expected but works. all provided software works with no errors.
### Screenshots
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/b9869cdd-4139-4338-a1b1-9d0e141e2204)
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/384f8780-44e5-41ab-a8eb-1f5b00b35bf9)
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/acd12016-4a31-49b1-b8d3-1a973931f5ff)

# TL;DR
Plasma 5 is good old Plasma 5. Everything just works.

**Comment by s0me1newithhand7s** [comment]
# Issue 3 in tree #3
### Config 
- Oracle VM VBox Manager
- toggled `plasma6.enable` to `true`;
- https://nixos.wiki/ and https://search.nixos.org/packages?channel=unstable
### Result / Error
no build errors, no syntaxis errors, no toggle errors;
KDE6 much better than i've expected, still not as polished as old KDE5 (last screenshot). all provided software works with no errors.
### Screenshots
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/4d4f2927-60bf-4a74-b14b-1514a1b3c244)
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/5f6f9dc9-8d05-4c96-a471-77b32a2913d4)
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/4433b802-19cf-42a1-969b-7c932cd450c4)

# TL;DR
wow. Plasma 6 is awesome. i would prefer KDE6 over KDE5 if there will be choice.

**Comment by s0me1newithhand7s** [comment]
# Issue 3 in tree #3
### Config 
- Oracle VM VBox Manager
- toggled `xfce.enable` to `true`;
- https://nixos.wiki/ and https://search.nixos.org/packages?channel=unstable
### Result / Error
no build errors, no syntaxis errors, no toggle errors;
### Screenshots
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/e6944d03-1402-4b19-8961-45dab5c003d1)
![image](https://github.com/s0me1newithhand7s/myNixConf/assets/117505144/a993f2e3-9b54-4b90-b3f7-f8b7d44f206b)

# TL;DR
rat have been poisoned. Xfce4.. not anymore lightweight DE, sadly. anyways, that's all for now.

**Comment by s0me1newithhand7s** [comment]
## TODO 
achieved
## Issue
closed
